MAINT: Fix E275 test failures

A change in flake8 seems to mean that there are now a few cases which fail on E275 (missing whitespace after keyword):
```
skypy/pipeline/tests/test_skypy.py:61:11: E275 missing whitespace after keyword
skypy/pipeline/tests/test_skypy.py:81:15: E275 missing whitespace after keyword
skypy/pipeline/tests/test_skypy.py:86:15: E275 missing whitespace after keyword
skypy/pipeline/tests/test_skypy.py:90:15: E275 missing whitespace after keyword
skypy/pipeline/tests/test_skypy.py:93:11: E275 missing whitespace after keyword
skypy/pipeline/tests/test_skypy.py:103:11: E275 missing whitespace after keyword
skypy/pipeline/tests/test_skypy.py:109:11: E275 missing whitespace after keyword
skypy/pipeline/scripts/skypy.py:51:11: E275 missing whitespace after keyword
skypy/utils/photometry.py:4:12: E275 missing whitespace after keyword
```
I will fix these now.

--- a/skypy/pipeline/scripts/skypy.py
+++ b/skypy/pipeline/scripts/skypy.py
@@ -48,4 +48,4 @@ def main(args=None):
         logger.exception(e)
         raise SystemExit(2) from e
 
-    return(0)
+    return (0)

--- a/skypy/pipeline/scripts/skypy.py
+++ b/skypy/pipeline/scripts/skypy.py
@@ -48,4 +48,4 @@ def main(args=None):
         logger.exception(e)
         raise SystemExit(2) from e
 
-    return (0)
+    return 0

--- a/skypy/pipeline/tests/test_skypy.py
+++ b/skypy/pipeline/tests/test_skypy.py
@@ -58,7 +58,7 @@ def test_logging(capsys, tmp_path):
     output_filename = str(tmp_path / 'logging.fits')
     skypy.main([config_filename, output_filename])
     out, err = capsys.readouterr()
-    assert(not err)
+    assert (not err)
 
     # Run again with increased verbosity and capture log. Force an exception by
     # not using the "--overwrite" flag when the output file already exists.
@@ -78,19 +78,19 @@ def test_logging(capsys, tmp_path):
     # Check all jobs appear in the log
     for job in list(config) + list(tables) + columns:
         log_string = f"[INFO] skypy.pipeline: Generating {job}"
-        assert(log_string in err)
+        assert (log_string in err)
 
     # Check all functions appear in the log
     for f in functions:
         log_string = f"[INFO] skypy.pipeline: Calling {f.function.__name__}"
-        assert(log_string in err)
+        assert (log_string in err)
 
     # Check cosmology appears in the log
     if cosmology:
-        assert("[INFO] skypy.pipeline: Setting cosmology" in err)
+        assert ("[INFO] skypy.pipeline: Setting cosmology" in err)
 
     # Check writing output file is in the log
-    assert(f"[INFO] skypy: Writing {output_filename}" in err)
+    assert (f"[INFO] skypy: Writing {output_filename}" in err)
 
     # Check error for existing output file is in the log
     try:
@@ -100,10 +100,10 @@ def test_logging(capsys, tmp_path):
     except ImportError:
         # Fallback on old error message from astropy v4.x
         error_string = f"[ERROR] skypy: File {output_filename!r} already exists."
-    assert(error_string in err)
+    assert (error_string in err)
 
     # Run again with decreased verbosity and check the log is empty
     with pytest.raises(SystemExit):
         skypy.main([config_filename, output_filename, '-qq'])
     out, err = capsys.readouterr()
-    assert(not err)
+    assert (not err)

--- a/skypy/pipeline/tests/test_skypy.py
+++ b/skypy/pipeline/tests/test_skypy.py
@@ -58,7 +58,7 @@ def test_logging(capsys, tmp_path):
     output_filename = str(tmp_path / 'logging.fits')
     skypy.main([config_filename, output_filename])
     out, err = capsys.readouterr()
-    assert (not err)
+    assert not err
 
     # Run again with increased verbosity and capture log. Force an exception by
     # not using the "--overwrite" flag when the output file already exists.
@@ -78,19 +78,19 @@ def test_logging(capsys, tmp_path):
     # Check all jobs appear in the log
     for job in list(config) + list(tables) + columns:
         log_string = f"[INFO] skypy.pipeline: Generating {job}"
-        assert (log_string in err)
+        assert log_string in err
 
     # Check all functions appear in the log
     for f in functions:
         log_string = f"[INFO] skypy.pipeline: Calling {f.function.__name__}"
-        assert (log_string in err)
+        assert log_string in err
 
     # Check cosmology appears in the log
     if cosmology:
-        assert ("[INFO] skypy.pipeline: Setting cosmology" in err)
+        assert "[INFO] skypy.pipeline: Setting cosmology" in err
 
     # Check writing output file is in the log
-    assert (f"[INFO] skypy: Writing {output_filename}" in err)
+    assert f"[INFO] skypy: Writing {output_filename}" in err
 
     # Check error for existing output file is in the log
     try:
@@ -100,10 +100,10 @@ def test_logging(capsys, tmp_path):
     except ImportError:
         # Fallback on old error message from astropy v4.x
         error_string = f"[ERROR] skypy: File {output_filename!r} already exists."
-    assert (error_string in err)
+    assert error_string in err
 
     # Run again with decreased verbosity and check the log is empty
     with pytest.raises(SystemExit):
         skypy.main([config_filename, output_filename, '-qq'])
     out, err = capsys.readouterr()
-    assert (not err)
+    assert not err

--- a/skypy/utils/photometry.py
+++ b/skypy/utils/photometry.py
@@ -131,7 +131,7 @@ def mag_ab(wavelength, spectrum, filters, *, redshift=None, coefficients=None,
         u = u.reshape(u.shape + (1,)*(nd_s+nd_f))
         m = np.ascontiguousarray(m[n])
         m += u*dm[n]
-        del (dm, n, u)
+        del dm, n, u
 
     # combine spectra if asked to
     if coefficients is not None:

--- a/skypy/utils/photometry.py
+++ b/skypy/utils/photometry.py
@@ -131,7 +131,7 @@ def mag_ab(wavelength, spectrum, filters, *, redshift=None, coefficients=None,
         u = u.reshape(u.shape + (1,)*(nd_s+nd_f))
         m = np.ascontiguousarray(m[n])
         m += u*dm[n]
-        del(dm, n, u)
+        del (dm, n, u)
 
     # combine spectra if asked to
     if coefficients is not None:

